### PR TITLE
Feature/#27 scenario results

### LINKED ISSUE #27: results from each scenario in a Jupytor Notebook
|# | Größe | Einheit | Plot/Tabelle | Kommentar | Done? |
|-------|----------|-----------|-----------------|-----------------|-----------------|
|1 | LCOE (Electricity, Heat) | divers | table | TODO #35  |x |
|2 | CO2 Emissionen| divers | table | 1) absolute, 2) year X compared to SQ, 3) compared to 1990 | 1) x |
|3 | Required Area | divers | Geoplots | absolute, relative | x|
|4| Autarky (balance)  | % | Bar plot | rel suppy/demand,  | x |
|5| Autarky (time) | hours,  % | Bar plot | hours a month/year | x |
|6| energymix | MWh | Barplot, areaplot (+ table) | generatio per technology | x |
|7| CO2 emissions per technology (elec. + heat) | tCO2 | stacked bar plot (elec/heat)| full year, 2 bars, technology stacked | x |
|8| rel use of flexibility options | | bar plot / boxplots | Ausgespeicherte Energie (Speicher), Durchgelaufene Energie (DSM) | |
|9| line loadings | % | bar plot | Ein Wert je Leitung | |
|10| line loadings | % | box plot | Median, whisker(min/max), full year | |
|11|  | % | Gridmap | (optional), Einfärbung nach Auslastung, pandapower| |
|12| LCOE+LCOE | €/Mwh | stacked bar plot jeweils für LCOE und LCOH | Anteilig für je Technologie. Daten in `results_scns[scn_id]['results_t']["LCOE"]` zu finden |x |

### PR DESCRIPTION
**Pull request by nailend**
although the flexibility plots of are not done yet, I reccomend to close #27, merge and continue with smaller issues

### CODE REVIEW

**Comment by nailend** [comment]
Thanks for your annotation and explanation in the notebook. With this and Guidos note-box its getting somewhere publicly presentable. 👍 
1. thx
2. Done
3. moved legend to the left side
4. yes I will move them. I just kept them there, to show them to you :)
5. I added this as additional trace to the plot but I am somehow not really satisfied. Maybe you can have a look.
I calculated the relative value as the ratio of the sum of supply and demand and added it to the analysis.tool

File: windnode_abw/analysis/tools.py [hunk 1]
```diff
@@ -1211,8 +1211,12 @@ def results_tech(results_axlxt):
     # Calculate levelized cost of heat
     results["LCOH"] = results_axlxt["Total costs heat supply"].sum() / results_axlxt['Wärmenachfrage nach Gemeinde'].sum().sum()
 
-    return results
+    # Autarky
+    results["Autarky"] = results_axlxt["Autarky"].loc[:,['supply','demand']].sum(axis=0).rename("ABW")
```
[the remark this answers is not in the document]
Comment (nailend) [reply windnode_abw/analysis/tools.py:1215]: I know, its just faster to grasp, that I access the columns and then sum over the rows. Sometimes I prefere the explicit to the implicit method.

**Comment by nailend** [comment]
Wasnt able to fix the secondary yaxis in a proper way... 
I also would like to use the plolty chloropleth but it seems impossible to convert the shapely format into geojson properly. Always ended up in arty structures....
For now I acctually gave this up.

**Comment by nesnoj** [comment]
> Wasnt able to fix the secondary yaxis in a proper way...

Hmmm, you gonna give it another try or do your need support?

> I also would like to use the plolty chloropleth but it seems impossible to convert the shapely format into geojson properly. Always ended up in arty structures....
> For now I acctually gave this up.

Yes, please leave it.

**Comment by gplssm** [comment]
I had look through the notebook and have some remarks/questions

- cool stuff there!
- can we move all multi-scenario analyses to #64, please
- can focus on finalizing plots and work towards a merge? I would really like, if we move bigger tasks to separate issues, since this PR is blocking #77 and #78 
- Can we include really basic stuff like inst. capacity and generated electricity/heat for the whole region (helps to check plausibility)

**Comment by nesnoj** [comment]
Fully agree!

> * Can we include really basic stuff like inst. capacity and generated electricity/heat for the whole region (helps to check plausibility)

There're geoplots on this, but totals for ABW would be great.

**Comment by nesnoj** [comment]
@nailend As described in #93 (brought up by you in #76), the el. demand for heating needs to be included in the energy balance. This affects the energy balance plots in 4.1 and 4.2 - please include "Heating" here using
`results_scns[scenario]['flows_txaxt']['Stromnachfrage Wärme'].sum(level=2).sum(axis=1)`.

**Comment by nailend** [comment]
>     * Can we include really basic stuff like inst. capacity and generated electricity/heat for the whole region (helps to check plausibility)

moved this to a new issue

**Pull request merged**